qa/tasks/cephfs/test_volumes.py

```diff
@@ -6605,6 +6613,139 @@ def test_snapshot_getpath_on_retained_subvol_in_group(self):
                                                  sv_uuid, group_name)
         self.assertEqual(snap_path, exp_snap_path)
 
+    def get_snap_dir_name(self):
+        from .fuse_mount import FuseMount
+        from .kernel_mount import KernelMount
+
+        if isinstance(self.mount_a, KernelMount):
+            return self.mount_a.client_config.get('snapdirname', '.snap')
```

Comment:
why not use `self.mount_a.snapdirname`?

Reply:
It is available in KernelMount instances but not in FuseMount instances.